synth/base/validator.py

```diff
@@ -164,8 +162,6 @@ def run(self):
             bt.logging.debug(
                 str(print_exception(type(err), err, err.__traceback__))
             )
-            bt.logging.debug("Scheduling validator to stop.")
-            self.should_exit = True
```

Comment:
We don't use anymore should_exit. So let's go back to the origin version (from the official template)